var/spack/repos/builtin/packages/pythia8/package.py

```diff
@@ -58,6 +58,14 @@ class Pythia8(AutotoolsPackage):
         deprecated=True,
     )
 
+    variant(
+        "cxxstd",
+        default="11",
+        values=("11", "17", "20", "23"),
```

Comment:
Why is 14 missing? (Not that it's much needed)

```suggestion
        values=("11", "14", "17", "20", "23"),
```

Reply:
I didn't forget, I only left 11 because it seems to be the default so I didn't want to change the current behavior, otherwise I think starting at 17 would be fine. Building on top of newer ROOT versions already requires C++17, so I thought 14 is quite old. Feel free to add it if you want.